src/content/docs/en/guides/deploy/azion.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Deploy your Astro Site to Azion
+description: How to deploy your Astro site to the web using Azion.
+type: deploy
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+You can deploy your Astro project on [Azion](https://console.azion.com/), a platform for frontend developers to collaborate and deploy static (JAMstack) and SSR websites.
+
+This guide includes:
+
+- [How to Deploy through Azion Console Dashboard](#how-to-deploy-a-static-site-github)
+- [How to deploy using the Azion CLI](#how-to-deploy-a-site-using-azion-cli)
+
+### Prerequisites
+
+To get started, you will need:
+
+- An [Azion account](https://www.azion.com/). If you don’t have one, you can sign up for a free account.  
+- Your app code stored in a [GitHub](https://github.com/) repository.  
+- [Azion CLI](https://www.azion.com/en/documentation/products/azion-cli/overview/) installed for faster project setup and deployment.
```

Comment:
As a windows user, the CLI requirement is not applicable in my case (putting it out there, no action needed)

Reply:
I agree, i made commit to include a caution to warn the Users.